### ISSUE TITLE
Setting  `trait_name : .na` for studies in long format

### ISSUE THREAD

**Issue by ehwenk** [traitecoevo/austraits.build#507]
For studies in long format, all `traits` are automatically added to the metadata file, even information in the `trait names` column that isn't a supported trait. 

For studies in wide format, if you add a `trait` (ie. column header) to the metadata that isn't supported by AusTraits, you can simply set `trait_name : .na` and it is ignored. 

But if you use this same syntax for the unsupported traits in a long format study, those trait names are flagged as `unsupported trait` in the excluded data table. Can this be fixed?

**Comment by GaryTruong** [comment]
@ehwenk Can you provide example studies where this has occurred including the code you would use?

**Comment by ehwenk** [comment]
ANBG_2019 is a good example. There are a number of rows in the dataset that include data that should not be read in: the rows in the column 'trait' (trait_name) that are 'LengthSample' and 'WidthSample' have data on sample size, not trait values. With long datasets, when the metadata file is propagated it adds a section for each trait that is found in the column with trait names. With a long dataset, if you set  `trait_name: .na`  the data are still read in and then pushed to the `excluded_data` table as `unsupported` trait, instead of being ignored.
```
- var_in: WidthSample
  unit_in: .na
  trait_name: .na
  value_type: .na
  replicates: .na
  methods: number of seeds examined to determine mean seed width
```
With wide datasets you can set traits to `trait_name: .na` and they are ignored. For instance, in Apgaua_2017, for a trait not yet supported by AusTraits

```
- var_in: Vessel lumen area (um2)
  unit_in: um2
  trait_name: .na
  value_type: raw_value
  replicates: 1
  methods: We sectioned wood samples with a GSL1 microtome (Gaertner, Lucchinetti...
```
I know it would be possible to filter out specific trait values in custom R code, but I want a solution that lets you have traits that appear in the list but aren't yet mapped to AusTraits designated traits, just like for wide datasets. 

In this example it would be appropriate to filter out these values, but in other cases, there could be traits that we want to be able to initially set `trait_name: na` and later fill in the trait name as appropriate, and not have all the values in the `excluded_data` table. 

Basically data from long and wide datasets should be treated the same way.

**Comment by ehwenk** [comment]
closed with commit c9d5bf1f0fc6af07e353c436c58236230d625cab
